examples/clustermanager/simple/UnixDomainCM.jl

```diff
@@ -9,7 +9,7 @@ function launch(manager::UnixDomainCM, params::Dict, launched::Array, c::Conditi
     for i in 1:manager.np
         sockname = tempname()
         try
-            cmd = `$(params[:exename]) --worker=custom $(@__FILE__) worker $sockname`
```

Comment:
No. And I don't have the actual args printed either. The command would have been something like 

`home/bezanson/src/julia/usr/bin/julia --worker=custom /home/bezanson/src/julia/usr/share/doc/julia/examples/clustermanager/simple/UnixDomainCM.jl worker /tmp/juliaxHYvd3`